test/test_helper.rb

```diff
@@ -6,14 +6,15 @@
 require 'timecop'
 require 'minitest/autorun'
 require 'minitest/stub/const'
+require 'mocha/mini_test'
 
-require 'helpers/kubeclient_helper'
-require 'helpers/fixture_deploy_helper'
-require 'helpers/fixture_set'
-require 'helpers/fixture_sets/hello-cloud'
-
+Dir.glob(File.expand_path("../helpers/**/*.rb", __FILE__)).each { |file| require file }
 ENV["KUBECONFIG"] ||= "#{Dir.home}/.kube/config"
 
+Mocha::Configuration.prevent(:stubbing_method_unnecessarily)
```

Comment:
yep! I didn't know either until recently. IIRC was chatting with @sirupsen about whether to use rspec or minitest for this project, and he pointed it out to me.